services/basic_types.proto

```diff
@@ -79,9 +79,30 @@ message AccountID {
     int64 realmNum = 2;
 
     /**
-     * A nonnegative account number unique within its realm
+     * The account number unique within its realm which can be either a non-negative integer or an alias public key.
```

Comment:
Perhaps mention that for any `AccountID` object that appears on a query response, transaction record, or transaction receipt the `accountNum` will be populated while the `alias` will be empty.

Reply:
Added as suggested